bin/templates/scripts/cordova/lib/prepare.js

```diff
@@ -275,12 +275,54 @@ function handleOrientationSettings (platformConfig, infoPlist) {
     }
 }
 
+// Make sure only update properties from our target project
+function updateBuildPropertyLocal (proj, targetName, prop, value, build) {
+    try {
+        // Check if we have a valid target - during prepare we do not have it
+        var target = proj.pbxTargetByName(targetName);
+        if (target == null || target.buildConfigurationList == null) {
```

Comment:
```suggestion
        if (!target || !target.buildConfigurationList) {
```
If you want to use `== null`, make sure you use triple equals `=== null`

Reply:
I am working to much with typescript. I have forgotton all JS quirks :-)